ci/docker/base.dockerfile

```diff
@@ -0,0 +1,77 @@
+# Base dockerfile to build images used in Darma testing.
+
+ARG ARCH=amd64
+ARG BASE=ubuntu:22.04
+ARG SETUP_ID=${SETUP_ID:-"ubuntu-develop"}
+
+# Compiler
+ARG CC=${CC:-""}
+ARG CXX=${CXX:-""}
+ARG FC=${FC:-""}
+ARG GCOV=${GCOV:-""}
+
+# MPI
+ARG MPICH_CC=${MPICH_CC:-""}
+ARG MPICH_CXX=${MPICH_CXX:-""}
+
+# PATH
+ARG PATH_PREFIX=${PATH_PREFIX:-""}
+
+# Specific (Intel One API)
+ARG CMPLR_ROOT \
+    INTEL_LICENSE_FILE \
+    ONEAPI_ROOT \
+    TBBROOT \
+    \
+    CPATH=${CPATH:-""} \
+    INFOPATH=${INFOPATH:-""} \
+    LD_LIBRARY_PATH=${LD_LIBRARY_PATH:-""} \
+    LIBRARY_PATH=${LIBRARY_PATH:-""}
```

Comment:
I don't think this needs to be placed before the `FROM` line. Try to move them down to avoid repetition.

Reply:
ok I am trying that